bears/r/FormatRBear.py

```diff
@@ -31,10 +31,10 @@ class FormatRBear:
     @staticmethod
     @deprecate_settings(indent_size='tab_width')
     def create_arguments(filename, file, config_file,
-                         r_keep_comments: bool=True,
-                         r_keep_blank_lines: bool=True,
-                         r_braces_on_next_line: bool=False,
-                         r_use_arrows: bool=False,
+                         r_keep_comments: bool=None,
+                         r_keep_blank_lines: bool=None,
```

Comment:
oh just seeing, those default values can be kept actually :)
I `keep_comments` is `True`, it keeps comments, so it doesn't change anything. Same applies for `keep_blank_lines` :D